### PR TITLE
Feature/icelandic language support

### PR DESCRIPTION
**Pull request by Kristinn-Stefansson**
As a native Icelandic speaker I wanted to contribute support for the language. Converting numeric data to words was my main aspiration so localisations of ToWords and ToOrdinalWords was the core aim, with Dates, TimeSpans, Headings and Bytes then being straightforward enough to add along the way. Icelandic has a few nuances there that some other North Germanic languages have left behind, even though heavier levels of inflection including four cases and noun declension are not in the picture. Still, I hope the submitted implementation is both transparent and in line with the established code. I have run a copy of the build process to validate that all tests pass on the PR, and did some benchmarks on the side to try and make sure the performance compared favorably with a selection of other language implementations.

The PR checklist: 
 - [X] Implementation is clean
 - [X] Code adheres to the existing coding standards; e.g. no curlies for one-line blocks, no redundant empty lines between methods or code blocks, spaces rather than tabs, etc.
 - [X] No Code Analysis warnings
 - [X] There is proper unit test coverage
 - [ ] If the code is copied from StackOverflow (or a blog or OSS) full disclosure is included. That includes required license files and/or file headers explaining where the code came from with proper attribution
 - [X] There are very few or no comments (because comments shouldn't be needed if you write clean code)
 - [ ] Xml documentation is added/updated for the addition/change
 - [X] Your PR is (re)based on top of the latest commits from the `dev` branch (more info below)
 - [ ] Link to the issue(s) you're fixing from your PR description. Use `fixes #<the issue number>`
 - [ ] Readme is updated if you change an existing feature or add a new one
 - [X] Run either `build.cmd` or `build.ps1` and ensure there are no test failures

### CODE REVIEW

File: src/Humanizer/Properties/Resources.is.resx [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+    <value>fyrir {0} sekúndu</value>
+  </data>
+    <data name="DateHumanize_MultipleSecondsFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} sekúndur</value>
+  </data>
+  <data name="DateHumanize_MultipleSecondsFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} sekúndu</value>
+  </data>
+    <data name="DateHumanize_MultipleYearsAgo_Above20" xml:space="preserve">
+    <value>{0} years ago</value>
```
Comment (clairernovotny) [inline src/Humanizer/Properties/Resources.is.resx:322]: Is this correct?

File: src/Humanizer/Properties/Resources.is.resx [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+    <value>fyrir {0} mánuði</value>
+  </data>
+      <data name="DateHumanize_MultipleMonthsFromNow_Plural" xml:space="preserve">
+    <value>eftir {0} mánuði</value>
+  </data>
+  <data name="DateHumanize_MultipleMonthsFromNow_Singular" xml:space="preserve">
+    <value>eftir {0} mánuð</value>
+  </data>
+    <data name="DateHumanize_MultipleSecondsAgo_Above20" xml:space="preserve">
+    <value>{0} seconds ago</value>
```
Comment (clairernovotny) [inline src/Humanizer/Properties/Resources.is.resx:307]: Is this correct?

File: src/Humanizer/Properties/Resources.is.resx [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+    <value>{0} mínútur</value>
+  </data>
+  <data name="TimeSpanHumanize_MultipleSeconds_Plural" xml:space="preserve">
+    <value>{0} sekúndur</value>
+  </data>
+  <data name="TimeSpanHumanize_MultipleMilliseconds_Plural" xml:space="preserve">
+    <value>{0} millisekúndur</value>
+  </data>
+  <data name="DateHumanize_MultipleDaysAgo_Above20" xml:space="preserve">
+    <value>{0} days ago</value>
```
Comment (clairernovotny) [inline src/Humanizer/Properties/Resources.is.resx:253]: Is this correct?

**Review by clairernovotny — COMMENTED** [review]
This looks great, thank you! I noticed a few things in the ResX that did not appear to be translated; are those accurate?

File: src/Humanizer/Properties/Resources.is.resx (hunk 1 above)
Comment (Kristinn-Stefansson) [reply src/Humanizer/Properties/Resources.is.resx]: Thanks @clairernovotny, good catch. I originally wanted to trace their use and see if they would be caught in tests but then forgot to finish cleaning up. I have translated them, and pushed a new commit.

File: src/Humanizer/Properties/Resources.is.resx (hunk 2 above)
Comment (Kristinn-Stefansson) [reply src/Humanizer/Properties/Resources.is.resx]: See previous revolved comment.

**Comment by clairernovotny** [comment]
The one remaining piece is to add a reference to this language in the main metapackage, like this:
https://github.com/Humanizr/Humanizer/blob/main/NuSpecs/Humanizer.nuspec#L36

Once that's done, I can merge this and get a release out.

**Pull request merged**